Makefile

```diff
@@ -217,6 +217,24 @@ image-gcp:
 	@tar -C $(PWD)/build -czf $(PWD)/build/gcp.tar.gz disk.raw
 	@rm -rf $(PWD)/build/disk.raw
 
+.PHONY: image-vmware
+image-vmware:
+	@docker run --rm -v /dev:/dev -v $(PWD)/build:/out \
+		--privileged $(DOCKER_ARGS) \
+		autonomy/installer:$(TAG) \
+		install \
+		-r \
+		-p vmware \
+		-u guestinfo \
+		-e console=tty0 \
+		-e earlyprintk=ttyS0,115200
+	@docker run --rm -v /dev:/dev -v $(PWD)/build:/out \
+		--privileged $(DOCKER_ARGS) \
+		autonomy/installer:$(TAG) \
+		vmdk
+	@ovftool -o hack/installer/talos.vmx build/talos.ova
```

Comment:
How difficult would it be to bundle `ovftool` within the installer?

Reply:
On second thought... let's keep this as is. I don't want to deal with VMware's asinine way of doing things.